pkg/features/kube_features.go

```diff
@@ -255,6 +255,12 @@ const (
 	//
 	// Enable container log rotation for cri container runtime
 	CRIContainerLogRotation utilfeature.Feature = "CRIContainerLogRotation"
+
+	// owner: @krmayankk
+	// alpha: v1.10
+	//
+	// Disable runAsNonRootGroup
+	RunAsNonRootGroup utilfeature.Feature = "RunAsNonRootGroup"
```

Comment:
not sure it makes sense to single out this aspect of the RunAsGroup feature... if we give control over the group, I'd expect all of it to come in as a single feature... runAsNonRootGroup and runAsGroup both working, or neither working